server.ts

```diff
@@ -82,6 +83,32 @@ export async function createFastifyServer(deviceRepo: DeviceRepository, recordRe
         reply.redirect(`/device/${deviceKey}`);
     });
 
+    // This is Rob, who is a noob here (and the architect, go figure...)
+    // attempting to produce a batch of keys at once.
+    // This can be tested with:
+    // curl -H "Content-Type: application/x-www-form-urlencoded" -X POST http://127.0.0.1:8000/manykeys/spudboy/3
+    // And will return something like:
+    // {"keys":["6bXXFunVNY9gtsY47n1tgQ","2UsUGTNYNFWA4jsdWBFxgf","LyGoHQ7wYr9XG1oWMkgXCx"]}
+    // ...which creates 3 records and keys with the name "spudboy".
+    type NumberParam = { n: number,
+                         name:   string };
+    server.post<{ Params: NumberParam }>('/manykeys/:name/:n', async (request, reply) => {
```

Comment:
You should be able to  use GET here if you wish since you're passing all the request data in the URL